src/QueryStats.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace YdbPlatform\Ydb;
+
+class QueryStats
+{
+
+    private $query_phases;
+    protected $compilation = null;
+    protected $process_cpu_time_us = 0;
+    protected $query_plan = '';
+    protected $query_ast = '';
+    protected $total_duration_us = 0;
+    protected $total_cpu_time_us = 0;
+    public function __construct(\Ydb\TableStats\QueryStats $queryStats)
+    {
+        $this->compilation = $queryStats->getCompilation();
```

Comment:
internally you can save full $queryStats object - it will be simpler, then split it to many parts.